lib/github-changelog.php

```diff
@@ -73,11 +73,27 @@ function get_changelog_html( $pr ) {
 function parse_changelog_html( $changelog_html ) {
 	preg_match( '/<h3>(.*)<\/h3>/', $changelog_html, $matches );
```

Comment:
Is it possible for an element other than <h3> to have been used upstream? Or what if no headings were included?

Reply:
> Is it possible for an element other than <h3> to have been used upstream?

Technically the changelog portion of the PR description can contain anything, but we only ever parsed out `<h3>`s, so if anything else was used, it never became a title.

> Or what if no headings were included?

Good catch - turns out it would throw a PHP warning if that happened. I've fixed that and added a test case in e4273d466aed16520a73a2e1ee213f6167381ae3